pyproject.toml

```diff
@@ -52,6 +51,9 @@ dependencies    = [
     "voluptuous-serialize==2.5.0",
     "yarl==1.7.2",
 ]
+optional-dependencies.orjson = [
+    "orjson==3.7.7",
+]
```

Comment:
Our CI and build systems needs to take this into account.

If I'm correct, this will now cause to ship without orjson for our containers (which make up ~95% of our user base).